app/views/shared/_web_app.html.haml

```diff
@@ -1,17 +1,17 @@
 - content_for :header_tags do
   - if user_signed_in?
-    = preload_pack_asset 'features/compose.js', crossorigin: 'anonymous'
-    = preload_pack_asset 'features/home_timeline.js', crossorigin: 'anonymous'
-    = preload_pack_asset 'features/notifications.js', crossorigin: 'anonymous'
+    -# = preload_pack_asset 'features/compose.js', crossorigin: 'anonymous'
+    -# = preload_pack_asset 'features/home_timeline.js', crossorigin: 'anonymous'
+    -# = preload_pack_asset 'features/notifications.js', crossorigin: 'anonymous'
```

Comment:
My understanding is that these maybe could be deleted because Vite automatically preloads? (assuming the application was structured in a way to code-split effectively)

Reply:
That's correct. Any chunks that the entrypoint depends on would be preloaded (even with early hints, when available).

An exception would be dynamically imported scripts, which are not preloaded by default (but can be preloaded manually, and the information is available in the manifest).